.github/workflows/image-push.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read  
```

Comment:
The permission to read EIO stuff is missing

Reply:
the permissions are just below inside the `jobs.push-multiarch` section

Reply:
sorry, you are right!